labeling/internal/empty.go

```diff
@@ -1,21 +1,26 @@
 package internal
 
 import (
+	"errors"
+	"strings"
+
 	"github.com/CircleCI-Public/circleci-config/labeling/codebase"
 	"github.com/CircleCI-Public/circleci-config/labeling/labels"
 )
 
 var EmptyRepoRules = []labels.Rule{
 	func(c codebase.Codebase, ls labels.LabelSet) (label labels.Label, err error) {
 		label.Key = labels.EmptyRepo
+		fileCount := 0
 		_, err = c.FindFileMatching(func(path string) bool {
-			if path != "." {
-				return true
+			if path != "." && strings.TrimSpace(strings.ToLower(path)) != "readme.md" {
+				fileCount += 1
 			}
+
 			return false
```

Comment:
I had to always return `false` since `FindFileMatching` stops on the first match and I didnt want to risk a scenario where `README.md` might be the first but not the only file.

Reply:
~ Couldn't we return false on that match, like https://github.com/CircleCI-Public/circleci-config/compare/detect-empty?expand=1#diff-c73a0ab923c70071befbaf91019c98b96d1bfa85b37e478024e9904b4ae49b04R13?

Reply:
Oh yeah good point I simplified the function to return false on "." and "readme" but return true on anything ( stop early if we find a file )